### PR TITLE
Add metadata for big maps and related content

### LINKED ISSUE #696: Update metadata to export big maps entities
The metadata configuration needs sections to expose big maps information to conseil's endpoints

### PR DESCRIPTION
**Pull request by ivanopagano**
fixes #696, #697 

Adding metadata definition for big_maps and big_map_contents entities on all available networks

### CODE REVIEW

**Comment by ivanopagano** [comment]
I left out the table connecting the originated contract with the map, since it only contained the external references.
Would that be needed?

**Comment by vishakh** [comment]
> I left out the table connecting the originated contract with the map, since it only contained the external references.
> Would that be needed?

@ivanopagano We definitely need to expose that table to our users.

File: src/main/resources/metadata/tezos.mainnet.conf [hunk 1]
```diff
@@ -338,9 +338,94 @@
...
+            entity: "accounts"
+            key: "account_id"
+          }
+        }
+      }
+    }
     blocks {
-      display-name-plural: "Blocks"
-      display-name: "Block"
+      display-name-plural: "blocks"
```
Comment (piotrkosecki) [inline src/main/resources/metadata/tezos.mainnet.conf]: why did you change it to lowercase only here?

Comment (ivanopagano) [reply src/main/resources/metadata/tezos.mainnet.conf]: I guess it must've been an unwilling change that slipped by... thanks

Comment (ivanopagano) [reply src/main/resources/metadata/tezos.mainnet.conf]: I think it might've been some semi-automated formatting that I didn't even expect to be there
